### DISCUSSION TITLE
Question on Dequeuing and Datachanges

### DISCUSSION THREAD

**Discussion by ctsaitsao** [FreeOpcUa/python-opcua#1417] [Q&A]
When a queue is being dequeued at once within a publish interval, how are values updated by the method `datachange_noficiation()`? Is it an "instant" change?

Thanks!

**Comment by AndreasHeine** [comment]
maybe this is what you asked for: 
https://github.com/FreeOpcUa/python-opcua/blob/67f15551884d7f11659d52483e7b932999ca3a75/opcua/common/subscription.py#L142

**Reply by ctsaitsao** [reply]
Yes, that's the code I was looking at. But how fast does it get called during a dequeuing process?

**Reply by AndreasHeine** [reply]
what is the issue? i mean you used the word "instant" ! so whats the issue you are facing?
